src/components/MyAccount/NewSettings/SaveCancelButtons.tsx

```diff
@@ -0,0 +1,43 @@
+import { ButtonGroup, Button } from "@nypl/design-system-react-components"
+
+type SaveCancelButtonProps = {
+  isDisabled: boolean
+  onCancel: () => void
+  onSave: () => void
+}
+
+const SaveCancelButtons = ({
+  isDisabled,
+  onCancel,
+  onSave,
+}: SaveCancelButtonProps) => {
+  return (
+    <ButtonGroup
+      display="flex"
+      flexDir={{ base: "column-reverse", lg: "row" }}
+      justifySelf={{ base: "unset", lg: "flex-end" }}
+      marginLeft={{ base: "unset", md: "l", lg: "auto" }}
+      marginTop={{ base: "s", lg: "unset" }}
+    >
+      <Button
+        sx={{ marginLeft: { base: "unset", lg: "xxl" } }}
+        id="cancel-button"
+        buttonType="secondary"
+        onClick={onCancel}
+      >
+        Cancel
+      </Button>
+      <Button
+        id="save-button"
+        isDisabled={isDisabled}
+        buttonType="primary"
+        minWidth={"123px"}
```

Comment:
This seems oddly specific. Was the width too small before?

Reply:
Yes i think it shouldn't stack

Reply:
I think you should be able to pass `buttonWidth="full"` in `ButtonGroup` for it to expand and not stack. Try that.